lib/components/Attachments/index.jsx

```diff
@@ -151,8 +152,9 @@ const Attachments = (
           />
         )}
         <input
-          multiple
+          accept={config?.allowedFileTypes?.join(",")}
```

Comment:
```suggestion
          accept={config.allowedFileTypes?.join(",")}
```
`config` will be non-nullable. It should at least be an empty object.